### ISSUE TITLE
Please give me some guide for "Paper and Code loss name matching"

### ISSUE THREAD

**Issue by dedoogong** [lolrudy/GPV_Pose#5]
Hello. I feel quite confused while trying to read your code.
I think 
3.1 L_Basic_rc     ==  cal_loss_R_con in fsnet_loss,py   << sure by k1 value
3.2 L_Basic_Sym == prop_sym_matching_loss in prop_loss.py  
3.3 L_Basic_pc    == cal_recon_loss_point in recon_loss.py  << sure by k2 value
3.4 L_PC_(R,t)  ==  cal_geo_loss_point in geometry_loss.py 
      L_PC_(s)    ==   cal_geo_loss_face in geometry_loss.py 
3.5 L_BB_(R,t,s)  ==  ??

But, maybe I'm wrong. 

Plus, can you tell me how did you get the specific value(13.7, 303.5)??
Please give me some advice~!

Thank you!

**Comment by lolrudy** [comment]
We select the two values emprically and we did not do experiments to confirm that it's optimal.
3.1 L_Basic_rc == cal_loss_R_con in fsnet_loss,py << sure by k1 value
3.2 L_Basic_Sym == prop_sym_matching_loss in prop_loss.py
3.3 L_Basic_pc == cal_recon_loss_point in recon_loss.py << sure by k2 value
3.4 L_PC_(R,t) == prop_point_matching_loss in prop_loss.py
Since L_PC_(s) is not really useful (also indicated in the paper), we remove the loss term and transform it into a pre-processing procedure. You can find it in the update.
3.5 L_BB_(R,t,s) == cal_recon_loss_vote in recon_loss.py

**Comment by awfniewf** [comment]
Hello, I have some questions about the loss function. About your code ：
##
 face x +
                f_n_xplus = f_n_now[:, 1, :]  # nn x 3
                res_xplus = torch.mean(1.0 - torch.mm(f_n_xplus, gt_r_x.view(3, 1)))
##
, which formula corresponds to the paper and what is its meaning? I hope you can answer, thank you.
